feat: support scroll to center to single element and rename `setScrollToContent` (#3482)
excalidraw/excalidraw: src/packages/excalidraw/CHANGELOG.md

### Features

> - #### BREAKING CHANGE
> 
>   - Renamed `setScrollToContent` to [`scrollToContent`](https://github.com/excalidraw/excalidraw/blob/master/src/components/App.tsx#L265). It now also supports supplying just a single Excalidraw element, or not supplying any argument at all which will default to current elements on the scene [#3482](https://github.com/excalidraw/excalidraw/pull/3482).
> 

Reviewer: ```suggestion
-  Support scrolling to center to a single Excalidraw element, or not supplying any argument at all which will default to current elements on the scene [#3482](https://github.com/excalidraw/excalidraw/pull/3482).
#### BREAKING CHANGE
  - Renamed `setScrollToContent` to [`scrollToContent`](https://github.com/excalidraw/excalidraw/blob/master/src/components/App.tsx#L265)..

```
So we mention feature first & only breaking change under breaking change.
